x-pack/legacy/plugins/ml/public/application/data_frame_analytics/common/analytics.ts

```diff
@@ -74,13 +75,26 @@ export interface RegressionEvaluateResponse {
   };
 }
 
+export interface ClassificationEvaluateResponse {
+  classification: {
+    multiclass_confusion_matrix: {
+      confusion_matrix: Array<{
```

Comment:
Suggest to create an interface `ConfusionMatrix` and use it here like `confusion_matrix: ConfusionMatrix[]`;